pkg/util/config/feature_gate.go

```diff
@@ -37,6 +37,8 @@ const (
 	// values override the default set by allAlpha.
 	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
 	allAlpha = "allAlpha"
+
+	externalTrafficLocalOnly = "externalTrafficLocalOnly"
```

Comment:
@thockin, aapproach sgtm, currently reviewing your pr. For this pr, just checking the feature-specific flag is sufficient. We'll make AllAlpha set defaults for other feature flags automatically.